### PR TITLE
we need to use truststore in the agent security realm now

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

**Comment by josejulio** [comment]
LGTM, I don't recall anything else that needs to be changed.

File: src/main/jbake/content/hawkular-services/docs/installation-guide/secure-comm.adoc [hunk 1]
```diff
@@ -72,6 +72,13 @@ First, you must add a security-realm in standalone/configuration/standalone.xml:
...
       </server-identities>
     </security-realm>
+    <security-realm name="HawkularAgentRealm">
+      <server-identities>
+        <ssl>
+          <truststore path="hawkular.keystore" relative-to="jboss.server.config.dir" keystore-password="hawkular" />
+        </ssl>
+      </server-identities>
+    </security-realm>
 ----
```
Comment (jkremser) [inline src/main/jbake/content/hawkular-services/docs/installation-guide/secure-comm.adoc]: This is what I've [added](https://github.com/hawkular/hawkular-services/pull/133/files#diff-e1e40939b471a17e473d9220afb7128bR41) and it worked

```xml
<security-realm>
  <authentication>
    <truststore path="hawkular.keystore" relative-to="jboss.server.config.dir" keystore-password="hawkular"/>
  </authentication>
</security-realm>
```

If the above code does the similar thing, then lgtm

**Review by jkremser — APPROVED** [review]

File: src/main/jbake/content/hawkular-services/docs/installation-guide/secure-comm.adoc (hunk 1 above)
Comment (josejulio) [reply src/main/jbake/content/hawkular-services/docs/installation-guide/secure-comm.adoc]: Actually i don't think is similar, Truststore must be placed in authentication node as far i remember.

**Comment by jmazzitelli** [comment]
I fixed what @Jiri-Kremser noticed. Uses authentication rather than server-identities in the security realm

**Pull request merged**